src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs

```diff
@@ -31,16 +31,27 @@ public override ProcessingPriority GetProcessingPriority(FileAndType file)
             switch (file.Type)
             {
                 case DocumentType.Article:
-                    if (".yml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase))
+                    if (".yml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase) ||
+                        ".yaml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase))
                     {
-                        return ProcessingPriority.Normal;
+                        var mime = YamlMime.ReadMime(Path.Combine(file.BaseDir, file.File));
```

Comment:
now the mime must start with "YamlMime:"

---
In reply to: [73289646](https://github.com/dotnet/docfx/pull/562#discussion_r73289646) [](ancestors = 73289646)